src/rhombus.song.js

```diff
@@ -205,7 +205,7 @@
       for (var instIdIdx in instruments._slots) {
         var instId = instruments._slots[instIdIdx];
         var inst = instruments._map[instId];
-        r.addInstrument(inst.type, inst.params, +instId, instIdIdx);
+        r.addInstrument(inst._type, inst._params, +instId, instIdIdx);
```

Comment:
Oh, the reason I didn't catch this was because the import/export tests I was doing involved the default instrument with default settings... so during import the instrument was being instantiated with the defaults (because the parameters here were undefined), which just happened to match the old parameters. Good catch.